### ISSUE TITLE
@javax.persistence.Entity mocking broken

### ISSUE THREAD

**Issue by Milbor** [jmockit/jmockit1#369]
1.29
java.lang.IllegalArgumentException: Invalid mocking of @Entity-annotated class
was fine before

```
public class SpikeTest {
    @Injectable
    private Codebook codebook;

    @javax.persistence.Entity
    public static class Codebook {
    }

    @Test
    public void test() throws Exception {
    }
}
```

**Comment by Milbor** [comment]
Duplicate of what? I don't see any linked issue.

**Comment by rliesenfeld** [comment]
Fixed together with changes for #326.
